documentation/docs/reference/dashboards/dashboard-mongodb-cluster-summary.md

```diff
@@ -2,60 +2,239 @@
+
+### Top Hottest Collections by Write
+
+Displays the five MongoDB collections with the highest write activity. It shows the number of write operations (inserts, updates, and deletes) per second for each collection, aggregated across your selected environment and cluster. 
+
+The bar gauge visualization helps quickly identify which collections are experiencing the most write activity, useful for performance monitoring and resource planning. Data is updated regularly to reflect recent changes in write patterns.
+
+### Operations Per Shard
+
+Shows the total number of operations per second for each shard in your MongoDB cluster. 
+
+It combines all types of operations (queries, inserts, updates, deletes, and getmore) into a single metric for each shard. The stacked area chart allows you to see both individual shard activity and total cluster activity over time. 
+
+This visualization helps you monitor the distribution of workload across shards and identify any imbalances or unusual patterns in operation rates.
+
+### MongoDB Versions
+
+Displays the current MongoDB version for each service in your cluster.
+
+This information helps you quickly identify which version of MongoDB is running on each service, ensuring all parts of your cluster are using consistent and up-to-date software.
+
+Use this to track version differences across your MongoDB deployment and plan upgrades as needed.
+
+## Current Topology
+
+Displays a visual representation of your MongoDB sharded cluster's architecture, organizing nodes into their respective functional groups. Each node is represented by a hexagonal icon with color-coding to indicate its role:
+
+- Green: Primary nodes or "UP" status for mongos routers
+- Yellow: Secondary nodes
+- Orange: Arbiter nodes (when present)
+- Red: Nodes in "DOWN" state
+
+This visualization provides a clear overview of your cluster's structure, making it easy to verify configuration, monitor health status, and ensure replication sets maintain the expected primary-secondary relationships.
+
+### Config Servers
+Displays config server nodes that store metadata and configuration settings for the entire cluster. These servers maintain the mapping of chunks to shards.
+
+### Mongos Routers
+Shows router instances that direct queries to appropriate shards. These routers process queries from the application layer and determine the location of data within the sharded cluster.
+
+### Shards
+Groups data-bearing nodes by shard name (e.g., `Shard - rs1`, `Shard - rs2`). Each shard contains a subset of the sharded data.
+
+### Configuration for proper visualization
+
+When adding MongoDB instances through `pmm-admin add mongodb` command or via the PMM UI, use these settings to ensure MongoDB cluster elements (config servers, shard nodes, and mongos routers) appear in the correct dashboard sections:
+
+ - Use the same `--cluster` name for all components
+ - Add each component as a separate service
+ - Set proper `--replication-set` values:
```

Comment:
I believe we should remove `--replication-set`, leave only reminder to use `--cluster` and point to https://docs.percona.com/percona-monitoring-and-management/3/install-pmm/install-pmm-client/connect-database/mongodb.html#on-the-command-line